config/manifests/bases/nvidia-network-operator.clusterserviceversion.yaml

```diff
@@ -64,4 +74,12 @@ spec:
   provider:
     name: NVIDIA
     url: https://github.com/Mellanox/network-operator/
+  relatedImages:
+  - image: docker.io/mellanox/mofed-5.5-1.0.3.2@sha256:5376561d6306d7db16860bd6c465c09a2ba2694965826aa043354d29b03f5221
+    name: mofed
+  - image: ghcr.io/k8snetworkplumbingwg/sriov-network-device-plugin@sha256:16a53286fecdb1e587d3c4c042078974674c3e86c9e98d7dae282f6eb4ee2d8c
+    name: sriov-network-device-plugin
+  - image: nvcr.io/nvidia/cloud-native/k8s-rdma-shared-dev-plugin@sha256:b25c9aa266ba8fd4a21633509b4118664d4b0a52d0d96781011101a90a559414
+    name: rdma-shared-device-plugin
+  replaces: nvidia-network-operator.v1.0.0
   version: 0.0.0
```

Comment:
version here gets replaced when creating the bundle ?

Reply:
This file is the template, so the version will be change when running `make bundle`.

For example:
`DEFAULT_CHANNEL=v1.1.0 CHANNELS=v1.1.0 VERSION=1.1.0 TAG=nvcr.io/nvidia/cloud-native/network-operator@sha256:17afa53f1cf3733c8d0cd282c565975ed5de3124dfc2b7c485ad12c97e51c251 make bundle`